ouroboros-consensus-cardano/src/unstable-cardano-testlib/Test/Consensus/Cardano/ProtocolInfo.hs

```diff
@@ -214,10 +214,10 @@ mkSimpleTestProtocolInfo
 --
 mkTestProtocolInfo ::
      forall m c
-   . (CardanoHardForkConstraints c, IOLike m, c ~ StandardCrypto)
+   . (CardanoHardForkConstraints c, IOLike m) --, c ~ StandardCrypto)
```

Comment:
This comment can be removed.

Reply:
Oops I overlooked this one. I will remove it.